internal/venice-consumer/src/main/java/com/linkedin/venice/client/consumer/ReplicationMetadataSchemaRepository.java

```diff
@@ -0,0 +1,38 @@
+package com.linkedin.venice.client.consumer;
+
+import com.linkedin.venice.controllerapi.ControllerClient;
+import com.linkedin.venice.controllerapi.MultiSchemaResponse;
+import com.linkedin.venice.controllerapi.MultiSchemaResponse.Schema;
+import com.linkedin.venice.exceptions.VeniceException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class ReplicationMetadataSchemaRepository {
```

Comment:
Honestly, this is a useful class if we ever decide we need to implement session consistency in Davinci/thin/fast client.  Keep it in mind.

Reply:
So you think we should move it out from this package to let it shared?